backend/users/middleware/auth_middleware.py

```
@@ -39,12 +43,13 @@ async def __call__(self, scope, receive, send):
 		decoded_query_string = query_string.decode()
 
 		access_token = parse_qs(decoded_query_string)["token"][0]
-		
 
 		if access_token == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)